docs/tutorials/pytorch2onnx.md

```diff
@@ -57,21 +59,22 @@ python tools/pytorch2onnx.py \
     --output-file checkpoints/resnet/resnet18_b16x8_cifar10.onnx \
     --dynamic-shape \
     --show \
+    --simplify \
```

Comment:
Please verify whether `--dynamic-shape` and `--simplify` could work together.

Reply:
`--dynamic-shape` and `--simplify` could work together.